src/jit/gentree.cpp

```diff
@@ -2096,19 +2097,9 @@ unsigned Compiler::gtHashValue(GenTree* tree)
                     hash += static_cast<unsigned>(tree->gtAddrMode.Offset() << 3) + tree->gtAddrMode.gtScale;
                     break;
 
-                case GT_BLK:
                 case GT_STORE_BLK:
-                    hash += tree->AsBlk()->GetLayout()->GetSize();
-                    break;
-
-                case GT_OBJ:
                 case GT_STORE_OBJ:
-                    hash ^= PtrToUlong(tree->AsObj()->GetLayout());
-                    break;
-
-                case GT_DYN_BLK:
```

Comment:
These 2 are `GTK_SPECIAL`, not `GTK_BINOP`. And they're already handled in the `special` section.